.jbang/JabKitLauncher.java

```diff
@@ -30,6 +30,10 @@
 //DEPS io.github.darvil82:terminal-text-formatter:2.3.0c
 //DEPS info.picocli:picocli:4.7.7
 //DEPS com.github.ben-manes.caffeine:caffeine:3.2.3
+//DEPS com.fasterxml.jackson.core:jackson-annotations:2.20
+//DEPS tools.jackson.core:jackson-core:3.0.2
+//DEPS tools.jackson.core:jackson-databind:3.0.2
+//DEPS tools.jackson.dataformat:jackson-dataformat-yaml:3.0.2
```

Comment:
Maybe, this can be left out, because it will come in using the jablib SNAPSHOT from maven central.